src/clientdlg.cpp

```diff
@@ -573,7 +573,15 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     // chance and the update check is not time-critical at all)
     CHostAddress CentServerHostAddress;
 
-    if ( NetworkUtil().ParseNetworkAddress ( DEFAULT_SERVER_ADDRESS, CentServerHostAddress ) )
+    // Send the request to two servers for redundancy if either or both of them
+    // has a higher release version number, the reply will trigger the notification.
+
+    if ( NetworkUtil().ParseNetworkAddress ( UPDATECHECK1_ADDRESS, CentServerHostAddress ) )
```

Comment:
Any reason why you don't use an or (||) here?

```
if ( NetworkUtil().ParseNetworkAddress ( UPDATECHECK1_ADDRESS, CentServerHostAddress ) || NetworkUtil().ParseNetworkAddress ( UPDATECHECK2_ADDRESS, CentServerHostAddress )  )

Reply:
I thought about this, too, but this is not just a check for the return value of `ParseNetworkAddress()`, but the call populates `CentServerHostAddress`. If this was done in one go, the second call would overwrite the first one.
Of course, one could split this into CentServerHostAddress1 and CentServerHostAddress2 as well, but I guess the current code saves some bytes of memory. :)

Reply:
And we actually want to send requests to both servers. There’s no point waiting for the reply from one before sending the request to the other, and that would be a lot more complicated to do.

Reply:
And each test-and-send is independent of the other. They just happen to share the same temporary variable.

Reply:
Ok. Thanks for the explanation.